src/components/accounts/TransactionItem.tsx

```diff
@@ -0,0 +1,262 @@
+import { StyleSheet, View } from "react-native";
+
+import TimeAgo from 'react-timeago'
```

Comment:
Do we really need this library?

Reply:
I considered just writing something similar, but it does have capability to update the value as time moves on, e.g. will go from "less than a minute ago" to "1 minute ago".  Its quite short and pretty well maintained, using it definitely sped up implementation enought that it seemed worth it.